index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <h2>Welcome</h2>
```

Comment:
- [ ] Very nice by having the logo on your web page, it looks really similar to the Figma design. Just there is a little thing that we need to improve here. Have a look at that logo, it's inside a ` h2` tag, however, the logo in the header is actually something that can bring us somewhere by clicking on it. So for that, kindly change the logo wrapper tag, I mean the `h1` into `a` tag.
   -  For more information about semantic Html, check out this [Semantic Html](https://www.w3schools.com/html/html5_semantic_elements.asp)